shared/instance.go

```diff
@@ -83,6 +83,10 @@ var InstanceConfigKeysAny = map[string]func(value string) error{
 	"boot.stop.priority":         validate.Optional(validate.IsInt64),
 	"boot.host_shutdown_timeout": validate.Optional(validate.IsInt64),
 
+	"cloud-init.network-config": validate.IsAny,
```

Comment:
I tend to wrap these in `validate.Optional()` so that it makes it clear these are optional keys (even though `Any` would handle that now) so that in the future if we expand the validation we won't have to reason about whether the implicit optionality of `Any` was intentional or not :)

Reply:
+1